lib/main.dart

```diff
@@ -18,37 +19,7 @@ class MyApp extends StatelessWidget {
       theme: ThemeData(
         primarySwatch: Colors.blue,
       ),
-      home: const MyHomePage(title: 'リコメンだー'),
-    );
-  }
-}
-
-class MyHomePage extends StatefulWidget {
-  const MyHomePage({super.key, required this.title});
-
-  final String title;
-
-  @override
-  State<MyHomePage> createState() => _MyHomePageState();
-}
-
-class _MyHomePageState extends State<MyHomePage> {
-  @override
-  Widget build(BuildContext context) {
-    return Scaffold(
-      appBar: AppBar(
-        title: Text(widget.title),
-      ),
-      body: Center(
-        child: Column(
-          mainAxisAlignment: MainAxisAlignment.center,
-          children: const <Widget>[
-            Text(
-              'You have pushed the button this many times:',
-            ),
-          ],
-        ),
-      ),
+      home: const LaunchPage(title: 'リコメンだー'),
```

Comment:
```suggestion
      home: const LaunchPage(),
```
L22 の `title: 'リコメンだー'` の部分は削除しちゃってください！

Reply:
`title: 'リコメンだー'`を消すとエラーになってしまいました！
constを削除したら、
以下のエラーが出たんですが、どうしたらよいでしょう・・

`The named parameter 'title' is required, but there's no corresponding argument. (Documentation)  Try adding the required argument.`

<img width="1056" alt="スクリーンショット 2022-12-07 19 10 38" src="https://user-images.githubusercontent.com/59502423/206151571-d191cdf0-fda6-4d71-9d6e-0bd66885b1b1.png">

Reply:
@Mayumi-Nakabayashi 

> L7 の , required this.title の部分
L9 の final String title;

https://github.com/flutteruniv/recommender/pull/23#discussion_r1041965032 こちらのコメントの上記箇所を削除すると解消されますでしょうか！

Reply:
@y129may9th 
失礼いたしました…！こちら先にやったらエラー解消できました〜〜！！